tagbot/action/repo.py

```diff
@@ -179,8 +195,15 @@ def _registry_pr(self, version: str) -> Optional[PullRequest]:
             return None
         name = self._project("name")
         uuid = self._project("uuid")
+        url = self._registry_url
+        if not url:
+            raise InvalidProject("Could not find url of package in registry")
+        url_hash = hashlib.sha256(url.encode()).hexdigest()
+        url_hash_trunc = url_hash[:10]
```

Comment:
nitpick but I think this is unnecessary, since we already use `uuid[:8]` in the f-string below instead of creating another variable

Reply:
Sorry, which part is unnecessary? Do you mean just directly do this?
```
        url_hash_trunc = hashlib.sha256(url.encode()).hexdigest()[:10]
```

Reply:
Oh, I understand